### PR TITLE
Fix up Text sections

### PR DESCRIPTION
**Pull request by karimkawambwa**
##### What does this PR do?

Fix up stuff
Show how it works and what is blast fishing

##### What are the relevant Pivotal Tracker stories?

Landing Page

##### Screenshots?

Screen Shot 2019-06-10 at 12 08 59 PM
Screen Shot 2019-06-10 at 12 09 31 PM
Screen Shot 2019-06-10 at 12 09 41 PM
Screen Shot 2019-06-10 at 12 08 51 PM

### CODE REVIEW

File: src/components/Line.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core';
+
+const styles = {
+  root: {
+    width: '80px',
+    height: '2px',
+    background: 'lightgrey',
+    marginTop: '20px',
+    borderLeft: 'solid #023256 20px'
```
Comment (kilemensi) [inline src/components/Line.js:11]: I think these are mobile screens. For desktop, they should be `75px/0.7px` and border of `22.5px` wide (and `1.5px` height)

**Review by kilemensi — APPROVED** [review]
:+1: @karimkawambwa.


--------------------------------------------------------------

The image _and_ the text after the image don't match the design. Is this going to be fixed in a separate PR?
Screenshot_2019-06-10 SeaSensors sensors AFRICA
vs
Screenshot from 2019-06-10 14-40-59

File: src/components/Line.js (hunk 1 above)
Comment (karimkawambwa) [reply src/components/Line.js:11]: Thats not the text section 

Thats addressed here https://github.com/CodeForAfricaLabs/sensors.AFRICA-SeaSensors/pull/34

**Review by KhadijaMahanga — APPROVED** [review]

**Review by nerdlet — APPROVED** [review]

File: src/components/Line.js (hunk 1 above)
Comment (karimkawambwa) [reply src/components/Line.js:11]: Will address this on another style fix

**Pull request merged**